airbyte-integrations/connectors/source-google-ads/metadata.yaml

```
@@ -38,6 +38,9 @@ data:
       - display_topics_performance_report
   releases:
     breakingChanges:
+      2.0.0:
+        message: This release includes enhancements to the default schemas and updates to stream names. Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.
```

Lets make the why a little more visible

```suggestion
        message: This release updates the Source Google Ads connector so that its default streams and stream names match the related resources in Google Ads API. Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.
```